src/displayapp/widgets/PageIndicator.cpp

```diff
@@ -0,0 +1,31 @@
+#include "displayapp/widgets/PageIndicator.h"
+
+using namespace Pinetime::Applications::Widgets;
+
+PageIndicator::PageIndicator(uint8_t nCurrentScreen, uint8_t nScreens) : nCurrentScreen {nCurrentScreen}, nScreens {nScreens} {
+}
+
+void PageIndicator::Create() {
```

Comment:
is the `Create()` function needed? Is there a reason not to move it to the constructor?

Reply:
It's to make it possible to draw the widget above other objects. If the widget is created during construction, it will always be below everything else created in the screen.